.gitignore

```
@@ -241,4 +241,5 @@ edm4hep/src/
 ## k4MarlinWrapper
 test/inputFiles/*.slcio
 test/gaudi_opts/testConverterConstants.py
-
+# Ignore root files generated in tests
+test/**/*root
```

Do we have any root files that need to be tracked? Otherwise we could also simply do
```suggestion
*.root
```

---

I would limit the ignore to just what is necessary, similarly to slcio files few lines above.